src/abi/program.rs

```diff
@@ -123,6 +123,8 @@ pub struct TypeMetadataDeclaration {
     pub components: Option<Vec<TypeApplication>>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub type_parameters: Option<Vec<MetadataTypeId>>,
+    #[serde(rename = "aliasOf", skip_serializing_if = "Option::is_none")]
```

Comment:
`rename = "aliasOf"` shouldn't be needed here, the `#[serde(rename_all = "camelCase")]` on `TypeMetadataDeclaration` already does it.
```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
```

Reply:
Will take it out from `TypeConcreteDeclaration` as well.